dev/NavigationView/NavigationView.cpp

```diff
@@ -2640,6 +2640,16 @@ void NavigationView::OnPropertyChanged(const winrt::DependencyPropertyChangedEve
     {
         UpdatePaneVisibility();
         UpdateVisualStateForDisplayModeGroup(DisplayMode());
+
+        if (!IsPaneVisible() && IsPaneOpen())
```

Comment:
Can you add some comments here? Otherwise it's hard to understand why you need this